nixos/modules/programs/polar.nix

```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.polar;
+in
+{
+  options = {
+    programs.polar = {
+      enable = mkEnableOption "Whether to install polar with udev rules.";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.polar ];
+    # See: https://github.com/cmaion/polar/blob/master/pkg/99-polar.rules
+    services.udev.extraRules = ''
```

Comment:
If your package installs the rule file into `lib/udev/rules.d`, then setting the option `services.udev.packages = [ pkgs.ploar ]` is sufficient. This avoids copying code and you can maybe consider to drop the module.

Reply:
Good idea! I'll make that change. It seems to be `etc/...`, not `lib/...` if I have understood correctly.

Reply:
The question now is: is it worth having a NixOS module to set those two small settings? Every additional NixOS module in the tree increases the evaluation time of nixos-rebuild.

Reply:
Good question, I don't know. Personally, I like that even such small settings are in NixOS. Reduces the amount of reading stuff here and there when trying to set up some program or service. But as the user would anyway need to figure out that they need to use `programs.polar` instead of `pkgs.polar`, the amount of things to figure out isn't that much reduced. But still, personally, I like having this kind of `programs` settings because I often search under `programs` when I'm installing an app to see if I should use that instead of `pkgs`. But I don't know about the general policy.

Reply:
How about this: we drop the module and instead mention `services.udev.packages = [ pkgs.polar ]` instead in longDescription in the package meta field?

Reply:
I now removed the NixOS module and added a comment in `longDescription`.